code-challenges.test.js

```diff
@@ -25,10 +25,30 @@ var secretCodeWord3 = "Eccentric"
 // Expected output: "3cc3ntr1c"
 
 
+describe('coded', () => {
+  it('converts a to 4, e to 3, i to 1, and o to 0', () => {
+    expect(coded(secretCodeWord1)).toEqual('L4ck4d41s1c4l')
+    expect(coded(secretCodeWord2)).toEqual('G0bbl3dyg00k')
+    expect(coded(secretCodeWord3)).toEqual('3cc3ntr1c')
+  })
+})
 
-// b) Create the function that makes the test pass.
 
+// b) Create the function that makes the test pass.
 
+//declare function that has an argument as a string
+const coded = (str) => {
+//split the string into an array and then map through it
+  return str.split('').map((e) => {
```

Comment:
This totally works, (e) threw me off a bit because I was thinking "event? what? ooooh".  "val" or "char" or "x" all are helpful for looking back at the code. But this is def a personal choice.